cmd/utils/cmd.go

```diff
@@ -170,6 +170,10 @@ func ImportChain(chain *core.BlockChain, fn string) error {
 	}
 	defer fh.Close()
 
+	if _, err := fh.Seek(18224628422, 0); err != nil {
+		panic(err)
+	}
```

Comment:
This code was to make the starting stage of the import be faster by skipping all blocks that might be already be applied in the state.

For our previous replay setup, that usually were a decent amount of blocks which wasted maybe 3 o 4 min at startup only "skipping" them.

I'm not sure how the new setup works on this regard -- the safest thing to do is simply first try to remove these lines and just avoid the run optimization. If the extra starting time is annoying, we could try to calculate this offset again but for the new setup.

Reply:
we should also keep this in a separate branch for replaying the old stuff, now we need to consistently replay at shapella, preferrably in CI

Reply:
Agree. The point was mostly to show that this trick (properly configured for the data) can save prob 5m of running time at the start. But that might not entirely matter if we do nightly runs in CI.

I'll remove it.